python/plot_porkchop.py

```diff
@@ -0,0 +1,176 @@
+
+times_of_flight = pd.read_sql_query("	SELECT DISTINCT time_of_flight 							  \
+										FROM lambert_scanner_results",							  \
+										database)
+departure_epochs = pd.read_sql_query("	SELECT DISTINCT departure_epoch 						  \
+										FROM lambert_scanner_results",							  \
+										database)-2400000.5 # change to Modified Julian Date
+transfer_delta_vs = pd.read_sql_query("	SELECT transfer_delta_v 								  \
+										FROM lambert_scanner_results 							  \
+										WHERE departure_object_id =" + str(a) + "				  \
+										and arrival_object_id =" + str(b),						  \
+										database)
+first_departure_epoch = departure_epochs['departure_epoch'][0]
+departure_epochs = departure_epochs - first_departure_epoch
+
+times_of_flight = times_of_flight / 86400
+
+z = np.array(transfer_delta_vs).reshape(len(departure_epochs), len(times_of_flight))
+x1, y1 = np.meshgrid(times_of_flight,departure_epochs)
+datapoints = np.size(z)
+failures = np.count_nonzero(np.isnan(z))
+
+# Add cutoff dV if user input requires to do so
+if config['transfer_deltaV_cutoff'] != 0:
+	cutoff = config['transfer_deltaV_cutoff']
+	for i in xrange(0,len(departure_epochs)):
+		for j in xrange(0,len(times_of_flight)):
+			if math.isnan(z[i][j]) == True:
+				z[i][j] = cutoff+1
+				# print z[i][j]
+			elif z[i][j] > cutoff:
+				z[i][j] =  cutoff
+
+# Plot porkchop plot
+cmap = plt.get_cmap(config['colormap'])
+fig=plt.figure()
+ax1 = fig.add_subplot(111)
+data = ax1.contourf(y1,x1,z,cmap=cmap)
+cbar = plt.colorbar(data, cmap=cmap)
+formatter = matplotlib.ticker.ScalarFormatter(useOffset=False)
+ax1.xaxis.set_major_formatter(formatter)
+ax1.yaxis.set_major_formatter(formatter)
+ax1.get_yaxis().set_tick_params(direction='out')
+ax1.get_xaxis().set_tick_params(direction='out')
+ax1.set_xlabel('Time since initial epoch [days] \n Initial departure epoch = ' + str(first_departure_epoch) + ' [mjd]', fontsize=13)
+ax1.set_ylabel('T$_{ToF}$ [days]', fontsize=13)
+cbar.ax.set_ylabel('Total transfer $\Delta V$ [km/s]', rotation=270, fontsize=13, labelpad=20)
+plt.ticklabel_format(style='sci', axis='x', scilimits=(0,0))
+
+if config['title'] == "True":
+	if config['transfer_deltaV_cutoff'] == 0:
+	 	plt.title("Porkchop plot of TLE elements " + str(a) + " to " + str(b) + 				  \
+	 			  ".\n Number of datapoints: " + str(datapoints) + " (" + 						  \
+	 			  str( len( departure_epochs ) ) + "x" + str(len(times_of_flight)) + 			  \
+	 			  "). Total failures: " + str(failures),		 								  \
+	 			  fontsize=10, y=1.02)
+	else:
+		plt.title("Porkchop plot of TLE elements " + str(a) + " to " + str(b) +				 	  \
+				  "\n with a cutoff $\Delta V$ of " + str(config['transfer_deltaV_cutoff']) + 	  \
+				  "\n Number of datapoints: " + str(datapoints) + " (" + 						  \
+				  str(len(departure_epochs)) + "x" + str(len(times_of_flight)) + ") " +			  \
+				  "Total failures: " + str(failures), 		  									  \
```

Comment:
arguments inside the plt.title function are not aligned properly, '\' missing at some places